MAINTAINERS.yml

```diff
@@ -2210,7 +2210,10 @@ West:
 "West project: hal_telink":
     status: maintained
     maintainers:
-        - yurvyn
+        - andy-liu-telink
+    collaborators:
+        - andriy-bilynskyy
+        - mishadesh
```

Comment:
It seems none of the users added above has any previous contributions to the Zephyr Project.

Although we do not have a concrete policy on the number of contributions required to qualify for the Collaborator and Maintainer roles, it does not make sense to add someone who has 0 contribution to the Zephyr Project as a collaborator or a maintainer.

I would suggest removing this change for now and letting these users add themselves to the `MAINTAINERS.yml` later when they have made some contributions upstream.

Reply:
These users are my colleagues from Telink Semiconductor. Yes, they did not contribute to Zephyr but they have good experience in Zephyr development and took part in Zephyr B91 Solution development.
They are going to replace me since I will not be available for some time.
So, could you please accept these changes?

Reply:
I am terminating my cooperation with Telink Semiconductor and my email will be deactivated soon.
So, please accept these changes.

Reply:
@yurvyn The normal process requires the maintainers and collaborators to have previous contributions to the Zephyr Project, but we can make an exception here so that the Telink platform does not get orphaned.

Please designate one (and only one) successor who will be maintaining this area and add him/her under `maintainers:`. Other collaborators can add themselves later through [the documented process](https://docs.zephyrproject.org/latest/project/project_roles.html#contributor) once they qualify.